tests/common/devices/sonic.py

```diff
@@ -1007,7 +1007,15 @@ def shutdown(self, ifname):
                 ifname: the interface to shutdown
         """
         logging.info("Shutting down {}".format(ifname))
-        return self.command("sudo config interface shutdown {}".format(ifname))
+
+        # if input ifname is alias, not interface name, need find out the interface name
+        ifs_status = self.get_interfaces_status()
```

Comment:
Shutting down each interface need to run "show interface status". This is not efficient enough. Can you optimize this?

Reply:
Sure, I reverted this commit and fix it by adding fanout_port_alias_to_name for sonic fanout device.
Please review the PR when free, thanks a lot.